tests/spec/w3c/headers-spec.js

```diff
@@ -73,6 +73,7 @@ describe("W3C — Headers", () => {
             mailto: "EMAIL",
             note: "NOTE",
             w3cid: "1234",
+            orcid: "https://orcid.org/0000-0002-1694-233X",
```

Comment:
I'd prefer to just do `0000-0002-1694-233X` rather than the full URL, like `w3cid`... What do you think?

Reply:
Do you mean in this particular test or in general? 

If you mean in general, the problem is that organization that defines ORCID mandates that the id be a URL. However the code supports both.

Reply:
I think the host part is a kind of namespace thing. It looks redundant because the field name is already `orcid` and requires double check for the host.

Reply:
So I agree it's redundant, thought their [docs state](https://support.orcid.org/hc/en-us/articles/360006897674) that:
 
> The ORCID iD is an https URI with a 16-digit number that is compatible with the ISO Standard (ISO 27729), also known as the International Standard Name Identifier (ISNI), e.g. https://orcid.org/0000-0001-2345-6789

My worry is that if we drop don't do the host part, people who use ORCID will complain that just the numeric components don't constitute a valid ORCID (technically correct)... I feel most people will be copy/pasting these URIs from the ORCID website into their ReSpec config. And then we will have other parts of the community using just the "16-digit number" part. The current design allows for both: error correction and we always end up with a conforming ORCID URL.

Reply:
Hmm... how about `orcidPath`, as the same documentation talks about the host and path?

Reply:
I feel this PR might be an overimplementation, doing full url and checksum, but if you feel good about it then it may be just okay.

Reply:
heh, yeah, I thought the same :) The checksum is a bit of overkill but it's tiny. The community (really a tiny set of editors in the Web Publishing WG) are the only ones wanting this and they are already using full URLs. For example, to original bug, and:
https://github.com/w3c/dxwg/blob/b32452e9fc003cfd2c1ddeebaa9355826313f62a/profilesont/config.js#L128

So I don't think we need to spend too many cycles on this. It's not something 99% of other folks will use.